enhancedsteam.js

```diff
@@ -5776,36 +5762,28 @@ function show_regional_pricing() {
 					});
 				}
 			});
-			if(subid_array.length>0){
-				$.each(countries,function(index,country){
-					switch (country) {
-						case "eu1":
-							cc="fr";
-							break;
-						case "eu2":
-							cc="it";
-							break;
-						default:
-							cc=country;
-							break;
-					}
-					$.each(subid_info,function(subid_index,package_info){
+
+			if (subid_array.length) {
+				$.each(countries, function(index, country) {
+					$.each(subid_info, function(subid_index, package_info) {
 						if (package_info["subid"] != 0) {
 							currency_deferred.push(
 								$.ajax({
-									url:api_url,
-									data:{
-										packageids:package_info["subid"],
-										cc:cc
+									url: api_url,
+									data: {
+										packageids: package_info["subid"],
+										// conversion for "eu2" should to be removed since "eu" regions were merged
```

Comment:
Before we do this we need a way to clean such settings from user's browser, a one time trigger when the extension is updated for example. See https://developer.chrome.com/extensions/runtime#event-onInstalled